redbeams/src/main/java/com/sequenceiq/redbeams/converter/v4/databaseserver/DatabaseServerConfigToDatabaseServerV4ResponseConverter.java

```diff
@@ -54,10 +54,13 @@ public DatabaseServerV4Response convert(DatabaseServerConfig source) {
             }
         } else if (source.getHost() != null && source.getPort() != null) {
             response.setStatus(Status.AVAILABLE);
+            response.setSslConfig(new SslConfigV4Response());
         } else {
             response.setStatus(Status.UNKNOWN);
+            response.setSslConfig(new SslConfigV4Response());
         }
 
```

Comment:
if (response.sslConfig == null) {
    response.setSslConfig(new SslConfigV4Response());
}
?

Reply:
Oh, good point, I will add it as a fallback default after the if/else if/else block.

Reply:
Done.